### PR TITLE
Removing deprecated AWS EC2 with Terraform and Digital Ocean App Platform V1 deployments

### PR DESCRIPTION
**Pull request by scottisloud**
This PR removes two deprecated deployment methods scheduled for deletion on 2023-09-14
* AWS EC2 with Terraform
* Digital Ocean App Platform V1

Both have been deprecated for longer than the documented period, and both have multiple viable alternatives. 

As a bonus, this PR improves the appearance of a `Note` callout box.

### CODE REVIEW

File: deprecated/README.md [hunk 1]
```diff
@@ -2,28 +2,30 @@
 
 This folder contains 1Password SCIM Bridge deployment methods that have been deprecated. At the time of deprecation, these deployments are still fully functional, but will no longer be updated.
 
-> 💡 **Note** that it is solely the _deployment method_ that is deprecated. Deprecating a deployment method is independent of the 1Password SCIM Bridge itself, or a specific version of the 1Password SCIM Bridge. For information about the latest version of 1Password SCIM Bridge, please see the [changelog](https://app-updates.agilebits.com/product_history/SCIM).
+> **Note**  
+> that it is solely the _deployment method_ that is deprecated. Deprecating a deployment method is independent of the 1Password SCIM Bridge itself, or a specific version of the 1Password SCIM Bridge. For information about the latest version of 1Password SCIM Bridge, please see the [changelog](https://app-updates.agilebits.com/product_history/SCIM).
```
Comment (plttn) [inline deprecated/README.md:6]: Minor nit, while we're here, any chance we can use the style guide approved phrasing? 
>... independent of the SCIM bridge itself, or a specific version of 1Pasword SCIM Bridge.

**Review by plttn — CHANGES_REQUESTED** [review]

File: deprecated/README.md (hunk 1 above)
Comment (scottisloud) [reply deprecated/README.md]: Done!

Comment (plttn) [inline deprecated/README.md]: One last nit: 

dangling sentence that is not capitalized due to note no longer being part of the copy.

**Review by plttn — CHANGES_REQUESTED** [review]
one last minor tweak

File: deprecated/README.md (hunk 1 above)
Comment (scottisloud) [reply deprecated/README.md]: fixed!

**Review by plttn — APPROVED** [review]
LGTM

**Review by ag-adampike — APPROVED** [review]
LGTM!

**Pull request merged**